card/mod/03_machines/set/type/scss.rb

```diff
@@ -4,12 +4,11 @@ def diff_args
   {:format=>:text}
 end  
 
-
 format do
   include Css::Format
   
   view :core do |args|
-    process_content compile_scss(_render_raw)
+    compile_scss(process_content _render_raw)
```

Comment:
I feel like there was a rationale for the previous order, but I can't remember what now.  Perhaps it was something like wanting to be able to click "add +image" in the middle of an html rendering of a css file?

At any rate, I think for now this is better.